specificatie/klanten/openapi.yaml

```diff
@@ -1046,8 +1084,16 @@ components:
           type: string
           maxLength: 3
           example: "nld"
-        adres:
-          $ref: '#/components/schemas/KlantAdres'
+        correspondentieadres:
+          oneOf:
+            - $ref: '#/components/schemas/Binnenlandsadres'
```

Comment:
Ken de conventies niet helemaal, maar zouden dit niet 'binnenlands_adres', 'buitenlands_adres' en 'postadres' moeten zijn?

Reply:
Goeie observatie, `binnenlandsadres` is niet één woord dus het moet CamelCase zijn: `BinnenlandsAdres`, etc. Het zijn schemadefinities dus ze moeten wel met een hoofdletter beginnen.